build-ci/third_party.rst

```diff
@@ -251,7 +251,12 @@ an annotated tag to your package repository using::
 
     git tag -a versionNumber -m "Some comment."
 
-The initial versionNumber should match the external package's version number.
+The initial ``versionNumber`` should match the external package's version
+number, where available; otherwise, it is acceptable to generate a version
+string based on e.g. an upstream git SHA1 or revision number. Note that
+version numbers should not start with a letter, as EUPS regards that as
+semantically significant.
```

Comment:
Do we want to suggest a form? `0.SHA1.lsstN` ? Otherwise people might look at the SHA1, see that theirs does start with a number and then just use it directly. I do wonder if we shouldn't have something like `0.1.SHA1`, then `0.2.SHA1` for the next release (or drop the leading 0.), so that we can look at the version numbers and see an actual progression (and we can sort the tags numerically!)